Makefile

```diff
@@ -233,6 +233,8 @@ promote-staging-manifest:
 	@cp -r manifest_staging/deploy .
 	@rm -rf chart
 	@cp -r manifest_staging/chart .
+	@helm package ./chart/gatekeeper-operator -d ./chart/gatekeeper-operator
+	@helm repo index ./chart/gatekeeper-operator --url https://raw.githubusercontent.com/open-policy-agent/gatekeeper/master/chart/gatekeeper-operator/
```

Comment:
I'm curious... is the helm repo guarded in any way by some sort of authentication?

Also, can this wait for the split helm repo? I have some renaming of the chart in that repo `gatekeeper-operator` is a misnomer.

Reply:
ah, I see what the repo commands are doing.

Reply:
No auth needed. It will package the chart as tgz archive and index.yaml inside GK repo. Once a user adds repo using `helm repo add` it will find the index.yaml and then pulls the chart from there.